### ISSUE TITLE
ACS Commons does not install in custom standalone installations

### ISSUE THREAD

**Issue by ehernandez-xk** [Adobe-Consulting-Services/acs-aem-commons#923]
Hi everyone

I want to install ACS Commons in the startup process of AEM, but with versions greater than 3.4.0 it is not possible to install it.

reference of Custom Standalone Install [Custom Standalone Install](https://docs.adobe.com/docs/en/aem/6-2/deploy/custom-standalone-install.html)

This is the process to reproduce the error

```
java -jar cq6.2-author-p4502.jar -unpack
mkdir -p crx-quickstart/install
cp acs-aem-commons-content-3.6.2-min.zip crx-quickstart/install/
./crx-quickstart/bin/start
```

When you go to the Package Manager (http://localhost:4502/crx/packmgr/index.jsp) the package appears at the bottom of the list and the status is installed but when you go to system/console/bundles the ACS bundle does not appear in the list

error.log
```
08.03.2017 10:32:04.532 *INFO* [OsgiInstallerImpl] org.apache.jackrabbit.vault.packaging.impl.JcrPackageImpl Creating snapshot for adobe/consulting/.snapshot:acs-aem-commons-content:3.6.2.
08.03.2017 10:32:04.626 *WARN* [OsgiInstallerImpl] org.apache.jackrabbit.oak.jcr.session.ItemImpl Item#refresh invokes Session#refresh!
08.03.2017 10:32:04.632 *INFO* [OsgiInstallerImpl] org.apache.jackrabbit.vault.packaging.impl.JcrPackageImpl Creating snapshot for adobe/consulting/.snapshot:acs-aem-commons-content:3.6.2 completed.
08.03.2017 10:32:04.632 *INFO* [OsgiInstallerImpl] org.apache.jackrabbit.vault.packaging.impl.ZipVaultPackage Extracting adobe/consulting:acs-aem-commons-content:3.6.2
08.03.2017 10:32:04.657 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXImporter Error during processing of /apps/acs-commons/authoring/audio-asset-finder: javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist
08.03.2017 10:32:04.657 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/audio-asset-finder (javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist)
08.03.2017 10:32:04.657 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/audio-asset-finder/audioController.js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.657 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/audio-asset-finder/js.txt (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.659 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXImporter Error during processing of /apps/acs-commons/authoring/dam-location: javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist
08.03.2017 10:32:04.659 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/dam-location (javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist)
08.03.2017 10:32:04.659 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/dam-location/js.txt (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.659 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/dam-location/reformat-degrees.js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.660 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXImporter Error during processing of /apps/acs-commons/authoring/editor-styles: javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist
08.03.2017 10:32:04.660 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/editor-styles (javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist)
08.03.2017 10:32:04.661 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/editor-styles/audio.css (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.661 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/editor-styles/css.txt (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.662 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXImporter Error during processing of /apps/acs-commons/authoring/shared-component-properties/classicui: javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist
08.03.2017 10:32:04.662 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/classicui (javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist)
08.03.2017 10:32:04.662 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/classicui/js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.662 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/classicui/js/shared-component-properties-classic.js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.662 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/classicui/js.txt (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.664 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXImporter Error during processing of /apps/acs-commons/authoring/shared-component-properties/touchui: javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist
08.03.2017 10:32:04.664 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/touchui (javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist)
08.03.2017 10:32:04.664 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/touchui/js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.664 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/touchui/js/shared-component-properties.js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.664 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/shared-component-properties/touchui/js.txt (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.665 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXImporter Error during processing of /apps/acs-commons/authoring/switchlayers: javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist
08.03.2017 10:32:04.665 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/switchlayers (javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist)
08.03.2017 10:32:04.665 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/switchlayers/js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.665 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/switchlayers/js/shortcuts.js (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.666 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/switchlayers/js.txt (java.lang.IllegalStateException: Parent node not found.)
08.03.2017 10:32:04.667 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXImporter Error during processing of /apps/acs-commons/authoring/vendor/jquery.fonticonpicker: javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist
08.03.2017 10:32:04.668 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/vendor/jquery.fonticonpicker (javax.jcr.nodetype.NoSuchNodeTypeException: Node type cq:ClientLibraryFolder does not exist)
08.03.2017 10:32:04.668 *ERROR* [OsgiInstallerImpl] org.apache.jackrabbit.vault.fs.io.Importer E /apps/acs-commons/authoring/vendor/jquery.fonticonpicker/css.txt (java.lang.IllegalStateException: Parent node not found.)
--
08.03.2017 10:32:08.036 *ERROR* [OsgiInstallerImpl] com.adobe.granite.installer.factory.packages.impl.PackageTransformer Error while processing install task of adobe/consulting:acs-aem-commons-content:3.6.2.
com.day.jcr.vault.packaging.PackageException: org.apache.jackrabbit.vault.packaging.PackageException: javax.jcr.nodetype.ConstraintViolationException: OakConstraint0001: /home/users[[rep:AuthorizableFolder, rep:AccessControllable]]: No matching definition found for child node system with effective type [nt:folder]
        at com.day.jcr.vault.packaging.impl.JrVltJcrPackageAdapter.install(JrVltJcrPackageAdapter.java:109)
        at com.adobe.granite.installer.factory.packages.impl.PackageTransformer$InstallPackageTask.execute(PackageTransformer.java:294)
        at org.apache.sling.installer.core.impl.OsgiInstallerImpl.doExecuteTasks(OsgiInstallerImpl.java:847)
        at org.apache.sling.installer.core.impl.OsgiInstallerImpl.executeTasks(OsgiInstallerImpl.java:689)
        at org.apache.sling.installer.core.impl.OsgiInstallerImpl.run(OsgiInstallerImpl.java:265)
        at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.jackrabbit.vault.packaging.PackageException: javax.jcr.nodetype.ConstraintViolationException: OakConstraint0001: /home/users[[rep:AuthorizableFolder, rep:AccessControllable]]: No matching definition found for child node system with effective type [nt:folder]
        at org.apache.jackrabbit.vault.packaging.impl.ZipVaultPackage.extract(ZipVaultPackage.java:239)
        at org.apache.jackrabbit.vault.packaging.impl.JcrPackageImpl.extract(JcrPackageImpl.java:423)
        at org.apache.jackrabbit.vault.packaging.impl.JcrPackageImpl.install(JcrPackageImpl.java:392)
        at com.day.jcr.vault.packaging.impl.JrVltJcrPackageAdapter.install(JrVltJcrPackageAdapter.java:107)
        ... 5 common frames omitted
Caused by: javax.jcr.nodetype.ConstraintViolationException: OakConstraint0001: /home/users[[rep:AuthorizableFolder, rep:AccessControllable]]: No matching definition found for child node system with effective type [nt:folder]
        at org.apache.jackrabbit.oak.api.CommitFailedException.asRepositoryException(CommitFailedException.java:225)
        at org.apache.jackrabbit.oak.api.CommitFailedException.asRepositoryException(CommitFailedException.java:212)
        at org.apache.jackrabbit.oak.jcr.delegate.SessionDelegate.newRepositoryException(SessionDelegate.java:670)
        at org.apache.jackrabbit.oak.jcr.delegate.SessionDelegate.save(SessionDelegate.java:496)
        at org.apache.jackrabbit.oak.jcr.session.SessionImpl$8.performVoid(SessionImpl.java:419)
        at org.apache.jackrabbit.oak.jcr.delegate.SessionDelegate.performVoid(SessionDelegate.java:274)
        at org.apache.jackrabbit.oak.jcr.session.SessionImpl.save(SessionImpl.java:416)
        at org.apache.jackrabbit.vault.fs.io.AutoSave.save(AutoSave.java:175)
        at org.apache.jackrabbit.vault.fs.io.Importer.run(Importer.java:416)
        at org.apache.jackrabbit.vault.packaging.impl.ZipVaultPackage.extract(ZipVaultPackage.java:234)
        ... 8 common frames omitted
Caused by: org.apache.jackrabbit.oak.api.CommitFailedException: OakConstraint0001: /home/users[[rep:AuthorizableFolder, rep:AccessControllable]]: No matching definition found for child node system with effective type [nt:folder]
        at org.apache.jackrabbit.oak.plugins.nodetype.TypeEditor.constraintViolation(TypeEditor.java:149)
        at org.apache.jackrabbit.oak.plugins.nodetype.TypeEditor.childNodeChanged(TypeEditor.java:257)
        at org.apache.jackrabbit.oak.plugins.nodetype.TypeEditor.childNodeAdded(TypeEditor.java:210)
```

**Comment by justinedelson** [comment]
Normally, this is handled by package dependencies, but given that we want to be able to support multiple AEM versions with a single package, I don't think this is easy to achieve. FWIW, this is not a documented installation mechanism.

**Comment by ehernandez-xk** [comment]
Thank you @justinedelson for your comment.

I think is the way on how the package is created in your build process, I made the following steps and the package is installed correctly on stand alone installation.

- Start the AEM 6.2 instance
- Install the ACS Commons package via Package Manager (I used 3.7.0)
- Select build the package
- Download the package
- Delete the AEM instance and remove crx-quickstart directory
- Follow the steps to install the package in custom install alone installation using the previously created package.
 
And works well.

**Comment by davidjgonzalez** [comment]
@ehernandez-xk can you unzip both packages (original acs commons and your rebuilt) and diff the output, and post it back here?

**Comment by ehernandez-xk** [comment]
Thank you @davidjgonzalez 

Please review if it is ok.

`diff -rq original/ fixed/`

Tell me if you need to see the diff of a specific file.

```
Only in original/META-INF: COPYRIGHT
Only in original/META-INF: LICENSE
Only in original/META-INF: MANIFEST.MF
Only in original/META-INF: NOTICE
Only in original/META-INF: maven
Files original/META-INF/vault/config.xml and fixed/META-INF/vault/config.xml differ
Files original/META-INF/vault/definition/.content.xml and fixed/META-INF/vault/definition/.content.xml differ
Files original/META-INF/vault/filter.xml and fixed/META-INF/vault/filter.xml differ
Only in fixed/META-INF/vault: nodetypes.cnd
Files original/META-INF/vault/properties.xml and fixed/META-INF/vault/properties.xml differ
Only in original/META-INF/vault: settings.xml
Only in fixed/jcr_root: .content.xml
Only in fixed/jcr_root/_oak_index: .content.xml
Files original/jcr_root/_oak_index/_rep_policy.xml and fixed/jcr_root/_oak_index/_rep_policy.xml differ
Files original/jcr_root/_rep_policy.xml and fixed/jcr_root/_rep_policy.xml differ
Only in fixed/jcr_root/apps: .content.xml
Files original/jcr_root/apps/_rep_policy.xml and fixed/jcr_root/apps/_rep_policy.xml differ
Files original/jcr_root/apps/acs-commons/authoring/shared-component-properties/classicui/.content.xml and fixed/jcr_root/apps/acs-commons/authoring/shared-component-properties/classicui/.content.xml differ
Files original/jcr_root/apps/acs-commons/authoring/shared-component-properties/touchui/.content.xml and fixed/jcr_root/apps/acs-commons/authoring/shared-component-properties/touchui/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/authoring/draggablelist/.content.xml and fixed/jcr_root/apps/acs-commons/components/authoring/draggablelist/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/content/definition-list/_cq_dialog/.content.xml and fixed/jcr_root/apps/acs-commons/components/content/definition-list/_cq_dialog/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/content/long-form-text/.content.xml and fixed/jcr_root/apps/acs-commons/components/content/long-form-text/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/content/long-form-text/_cq_editConfig.xml and fixed/jcr_root/apps/acs-commons/components/content/long-form-text/_cq_editConfig.xml differ
Files original/jcr_root/apps/acs-commons/components/content/long-form-text/dialog.xml and fixed/jcr_root/apps/acs-commons/components/content/long-form-text/dialog.xml differ
Files original/jcr_root/apps/acs-commons/components/content/long-form-text/long-form-text-parsys/.content.xml and fixed/jcr_root/apps/acs-commons/components/content/long-form-text/long-form-text-parsys/.content.xml differ
Only in fixed/jcr_root/apps/acs-commons/components/content/long-form-text/long-form-text-parsys: new
Files original/jcr_root/apps/acs-commons/components/content/named-transform-image/_cq_editConfig.xml and fixed/jcr_root/apps/acs-commons/components/content/named-transform-image/_cq_editConfig.xml differ
Files original/jcr_root/apps/acs-commons/components/content/named-transform-image/dialog.xml and fixed/jcr_root/apps/acs-commons/components/content/named-transform-image/dialog.xml differ
Files original/jcr_root/apps/acs-commons/components/content/sharethis-buttons/_cq_dialog/.content.xml and fixed/jcr_root/apps/acs-commons/components/content/sharethis-buttons/_cq_dialog/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/content/sharethis-counts/_cq_dialog/.content.xml and fixed/jcr_root/apps/acs-commons/components/content/sharethis-counts/_cq_dialog/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/app-page/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/app-page/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/app-page/clientlibs/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/app-page/clientlibs/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/audit-log-search/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/audit-log-search/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/exporters/users-to-csv/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/exporters/users-to-csv/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/jcr-compare/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/jcr-compare/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/jcr-compare/clientlibs/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/jcr-compare/clientlibs/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/oak-index-manager/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/oak-index-manager/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/packager/authorizable-packager/configuration/dialog.xml and fixed/jcr_root/apps/acs-commons/components/utilities/packager/authorizable-packager/configuration/dialog.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/configuration/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/configuration/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/configuration/_cq_editConfig.xml and fixed/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/configuration/_cq_editConfig.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/configuration/dialog.xml and fixed/jcr_root/apps/acs-commons/components/utilities/packager/query-packager/configuration/dialog.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/quickly/clientlibs/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/quickly/clientlibs/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/system-notifications/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/system-notifications/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/system-notifications/clientlibs/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/system-notifications/clientlibs/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/system-notifications/notification/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/system-notifications/notification/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/system-notifications/notification/clientlibs/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/system-notifications/notification/clientlibs/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/utilities/workflow-remover/.content.xml and fixed/jcr_root/apps/acs-commons/components/utilities/workflow-remover/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/.content.xml and fixed/jcr_root/apps/acs-commons/components/workflow/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/.content.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/activate/.content.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/activate/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/activate/_cq_editConfig.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/activate/_cq_editConfig.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/cache-invalidate/.content.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/cache-invalidate/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/cache-invalidate/_cq_editConfig.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/cache-invalidate/_cq_editConfig.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/deactivate/.content.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/deactivate/.content.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/deactivate/_cq_editConfig.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/deactivate/_cq_editConfig.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/dialog.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/dialog.xml differ
Files original/jcr_root/apps/acs-commons/components/workflow/select-agent/processargs.xml and fixed/jcr_root/apps/acs-commons/components/workflow/select-agent/processargs.xml differ
Files original/jcr_root/apps/acs-commons/config/org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended-acs-commons.xml and fixed/jcr_root/apps/acs-commons/config/org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended-acs-commons.xml differ
Files original/jcr_root/apps/acs-commons/templates/utilities/bulk-workflow-manager/.content.xml and fixed/jcr_root/apps/acs-commons/templates/utilities/bulk-workflow-manager/.content.xml differ
Files original/jcr_root/apps/acs-commons/templates/utilities/exporters/users-to-csv/.content.xml and fixed/jcr_root/apps/acs-commons/templates/utilities/exporters/users-to-csv/.content.xml differ
Files original/jcr_root/apps/acs-commons/templates/utilities/query-packager/.content.xml and fixed/jcr_root/apps/acs-commons/templates/utilities/query-packager/.content.xml differ
Files original/jcr_root/apps/acs-commons/templates/utilities/system-notifications/.content.xml and fixed/jcr_root/apps/acs-commons/templates/utilities/system-notifications/.content.xml differ
Files original/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/.content.xml and fixed/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/.content.xml differ
Only in fixed/jcr_root/apps/acs-commons/touchui-widgets/contextualpathbrowser: .context
Only in original/jcr_root/apps/acs-commons/touchui-widgets/contextualpathbrowser: .context.xml
Files original/jcr_root/apps/acs-commons/widgets/add-ons/wcm-views/.content.xml and fixed/jcr_root/apps/acs-commons/widgets/add-ons/wcm-views/.content.xml differ
Files original/jcr_root/apps/cq/core/content/.content.xml and fixed/jcr_root/apps/cq/core/content/.content.xml differ
Files original/jcr_root/apps/cq/core/content/nav/.content.xml and fixed/jcr_root/apps/cq/core/content/nav/.content.xml differ
Only in original/jcr_root/apps/cq/core/content/nav/tools: .content.xml
Files original/jcr_root/apps/cq/core/content/nav/tools/acs-commons/.content.xml and fixed/jcr_root/apps/cq/core/content/nav/tools/acs-commons/.content.xml differ
Files original/jcr_root/apps/cq/core/content/nav/tools/acs-commons/jcr-compare/.content.xml and fixed/jcr_root/apps/cq/core/content/nav/tools/acs-commons/jcr-compare/.content.xml differ
Only in fixed/jcr_root/content: .content.xml
Files original/jcr_root/content/_rep_policy.xml and fixed/jcr_root/content/_rep_policy.xml differ
Only in fixed/jcr_root/content/dam: .content.xml
Files original/jcr_root/content/dam/_rep_policy.xml and fixed/jcr_root/content/dam/_rep_policy.xml differ
Only in fixed/jcr_root/etc: .content.xml
Files original/jcr_root/etc/acs-commons/bulk-workflow-manager/.content.xml and fixed/jcr_root/etc/acs-commons/bulk-workflow-manager/.content.xml differ
Only in original/jcr_root/etc/acs-commons/bulk-workflow-manager: _rep_policy.xml
Files original/jcr_root/etc/acs-commons/jcr-compare/.content.xml and fixed/jcr_root/etc/acs-commons/jcr-compare/.content.xml differ
Files original/jcr_root/etc/acs-commons/lists/.content.xml and fixed/jcr_root/etc/acs-commons/lists/.content.xml differ
Files original/jcr_root/etc/acs-commons/notifications/.content.xml and fixed/jcr_root/etc/acs-commons/notifications/.content.xml differ
Only in original/jcr_root/etc/acs-commons/notifications: _rep_policy.xml
Files original/jcr_root/etc/acs-commons/workflow-remover/.content.xml and fixed/jcr_root/etc/acs-commons/workflow-remover/.content.xml differ
Only in fixed/jcr_root/etc/clientlibs: .content.xml
Files original/jcr_root/etc/clientlibs/acs-commons/angularCoral/.content.xml and fixed/jcr_root/etc/clientlibs/acs-commons/angularCoral/.content.xml differ
Files original/jcr_root/etc/clientlibs/acs-commons/vendor/lodash/.content.xml and fixed/jcr_root/etc/clientlibs/acs-commons/vendor/lodash/.content.xml differ
Files original/jcr_root/etc/cloudservices/.content.xml and fixed/jcr_root/etc/cloudservices/.content.xml differ
Files original/jcr_root/etc/cloudservices/dtm/.content.xml and fixed/jcr_root/etc/cloudservices/dtm/.content.xml differ
Files original/jcr_root/etc/cloudservices/dtm/_rep_policy.xml and fixed/jcr_root/etc/cloudservices/dtm/_rep_policy.xml differ
Files original/jcr_root/etc/cloudservices/sharethis/.content.xml and fixed/jcr_root/etc/cloudservices/sharethis/.content.xml differ
Files original/jcr_root/etc/cloudservices/sharethis/_rep_policy.xml and fixed/jcr_root/etc/cloudservices/sharethis/_rep_policy.xml differ
Files original/jcr_root/etc/cloudservices/typekit/.content.xml and fixed/jcr_root/etc/cloudservices/typekit/.content.xml differ
Files original/jcr_root/etc/cloudservices/typekit/_rep_policy.xml and fixed/jcr_root/etc/cloudservices/typekit/_rep_policy.xml differ
Files original/jcr_root/etc/dam/.content.xml and fixed/jcr_root/etc/dam/.content.xml differ
Files original/jcr_root/etc/dam/video/.content.xml and fixed/jcr_root/etc/dam/video/.content.xml differ
Files original/jcr_root/etc/dam/video/flacmono/.content.xml and fixed/jcr_root/etc/dam/video/flacmono/.content.xml differ
Only in fixed/jcr_root/etc/designs: .content.xml
Only in fixed/jcr_root/etc/notification: .content.xml
Only in fixed/jcr_root/etc/notification/email: .content.xml
Files original/jcr_root/etc/notification/email/_rep_policy.xml and fixed/jcr_root/etc/notification/email/_rep_policy.xml differ
Only in fixed/jcr_root/etc/workflow: .content.xml
Only in fixed/jcr_root/etc/workflow/instances: .content.xml
Files original/jcr_root/etc/workflow/instances/_rep_policy.xml and fixed/jcr_root/etc/workflow/instances/_rep_policy.xml differ
Only in fixed/jcr_root/home: .content.xml
Only in fixed/jcr_root/home/users: .content.xml
Files original/jcr_root/home/users/_rep_policy.xml and fixed/jcr_root/home/users/_rep_policy.xml differ
Only in fixed/jcr_root/home/users/system: .content.xml
```

**Comment by justinedelson** [comment]
resolved by #1111
